### PR TITLE
Can i have your number

### LINKED ISSUE #281: Mobile collection for international
Optionally, allow for international users to add their cell number

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
- Optionally collect phone number for UK & Brazil users. Closes #281.

For review: @angaither 

screen shot 2015-05-26 at 2 09 38 pm

### CODE REVIEW

File: resources/views/auth/form.blade.php [hunk 1]
```diff
@@ -11,6 +11,11 @@
     {!! Form::text('email', null, ['placeholder' => 'you@example.com']) !!}
 @endif
 
+@if(should_collect_international_phone())
+    {!! Form::label('phone', 'Cell Number (optional)') !!}
+    {!! Form::text('phone', null, ['placeholder' => '555-555-5555']) !!}
```
Comment (angaither) [inline resources/views/auth/form.blade.php]: should this be an `||` with the statement below? or do we want to make them feel welcome, and put things in the placeholder that fit their phone number styleeee?

Comment (DFurnes) [inline resources/views/auth/form.blade.php]: 1. I kept it separate so we can add the `(optional)` to the label.
2. Oh boyyyyyyy. I feel like that could get messy to maintain. Lemme make an issue & we can look into how easy translation in Laravel is...

Comment (DFurnes) [inline resources/views/auth/form.blade.php]: Never mind. Laravel FTW! :boom:

**Comment by angaither** [comment]
:elephant: :+1:

**Pull request merged**